backend/src/main/java/ru/hh/school/resource/EmployerFavoritesResource.java

```diff
@@ -0,0 +1,96 @@
+package ru.hh.school.resource;
+
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.hh.school.Verification;
+import ru.hh.school.dto.EmployerAddFavoritesDto;
+import ru.hh.school.dto.EmployerGetFavoritesDto;
+import ru.hh.school.dto.EmployerUpdateFavoritesDto;
+import ru.hh.school.service.EmployerFavoritesService;
+
+import javax.inject.Singleton;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static ru.hh.school.Params.*;
+
+@Path("/favorites/employer")
+@RequiredArgsConstructor
+@Singleton
+public class EmployerFavoritesResource {
+    private static final Logger logger = LoggerFactory.getLogger(EmployerFavoritesResource.class);
+    private final EmployerFavoritesService employerFavoritesService;
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response addEmployerToFavorites(EmployerAddFavoritesDto employer) {
+        logger.info("Invoked POST/favorites/employer with params: employer.id = {};", employer.getEmployerId());
+
+        if (employer.getEmployerId() <= 0)
+            throw new IllegalArgumentException("The employer.id is less or equal 0: " + employer.getEmployerId());
+
+        long result = employerFavoritesService.addEmployer(employer);
+        //return Response.status(Response.Status.CREATED).entity(json).build();
+        return Response.ok("Employer was added, id = " + result).build();
```

Comment:
Стоит аннотация `@Produces(MediaType.APPLICATION_JSON)`, но возвращается просто строка, а не json.
Я в таких случаях делаю что-то типа
```suggestion
        return Response.ok(Map.of("message", "Employer was added, id = " + result)).build();
```